txtorcon/torcontrolprotocol.py

```diff
@@ -18,6 +18,11 @@
 import types
 import base64
 
+try:
+    import stem.version as stem_version
+except ImportError:
+    stem_version = None
+
 DEFAULT_VALUE = 'DEFAULT'
```

Comment:
*perhaps* we should think about only trying the import if "use_stem=True" in TorControlProtocol. Mostly just thinking out loud here...Basically thinking longer-term about how to have stem-or-nor requirements.